example/issue_comment.go

```diff
@@ -131,12 +131,21 @@ func (h *PRCommentHandler) Handle(ctx context.Context, eventType, deliveryID str
 // extractProwJobURLFromCommentBody extracts the
 // Prow job's URL from the given PR comment's body
 func extractProwJobURLFromCommentBody(logger zerolog.Logger, commentBody string) (string, error) {
+	var prowJobURL string
+
 	r, _ := regexp.Compile(regexToFetchProwURL)
 	sliceOfMatchingString := r.FindStringSubmatch(commentBody)
 	if sliceOfMatchingString == nil {
 		return "", fmt.Errorf("regex string %s found no matches for the comment body: %s", regexToFetchProwURL, commentBody)
 	}
-	prowJobURL := sliceOfMatchingString[1]
+	for _, match := range sliceOfMatchingString {
+		if !strings.Contains(match, "images") && !strings.HasSuffix(match, ")") {
```

Comment:
what does the condition `!strings.HasSuffix(match, ")")` mean? i.e. why it's needed?

Reply:
The regex returns 2 matches per prow job, one with a closing parenthesis, and the other without it (as seen below at bottom right)
![image](https://github.com/redhat-appstudio-qe/ci-helper-app/assets/43273420/cdf3cab4-a2b1-4394-b7fd-735b3d6dec53)

And this condition makes sure that we filter out the one having ")" at the end.